src/st/rare/e_fountain_water.c

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "rare.h"
+
+#include "../are/e_fountain_water.h"
```

Comment:
This might be a Bad Idea... I am not sure whether this would be the best way to share code out of the non-inverted stage which is specific to the stage itself (rather than being a generalised shared entity stored in `./st`).

I can copy the code here instead of doing this if preferred